jetty-server/src/main/java/org/eclipse/jetty/server/handler/InetAccessHandler.java

```diff
@@ -123,23 +181,30 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     /**
      * Checks if specified address and request are allowed by current InetAddress rules.
      *
-     * @param address the inetAddress to check
-     * @param request the request to check
+     * @param addr     the inetAddress to check
+     * @param baseRequest the base request to check
+     * @param request     the HttpServletRequest request to check
      * @return true if inetAddress and request are allowed
      */
-    protected boolean isAllowed(InetAddress address, HttpServletRequest request)
+    protected boolean isAllowed(InetAddress addr, Request baseRequest, HttpServletRequest request)
     {
-        boolean allowed = _set.test(address);
+        String name = baseRequest.getHttpChannel().getConnector().getName();
         if (LOG.isDebugEnabled())
-            LOG.debug("{} {} {} for {}", this, allowed ? "allowed" : "denied", address, request);
-        return allowed;
+        {
+            Boolean allowedByName = _names.isIncludedAndNotExcluded(name);
+            Boolean allowedByAddr = _addrs.isIncludedAndNotExcluded(addr);
+            LOG.debug("{} allowedByName={} allowedByAddr={} for {}/{}", this, allowedByName, allowedByAddr, addr, request);
+        }
+        return IncludeExclude.and(_addrs, addr, _names, baseRequest.getHttpChannel().getConnector()::getName);
     }
 
     @Override
     public void dump(Appendable out, String indent) throws IOException
     {
         dumpObjects(out, indent,
-            DumpableCollection.from("included",_set.getIncluded()),
-            DumpableCollection.from("excluded",_set.getExcluded()));
+                Dumpable.labelled("included", _addrs.getIncluded()),
+                Dumpable.labelled("excluded", _addrs.getExcluded()),
+                Dumpable.labelled("includedConnectorNames", _names.getIncluded()),
+                Dumpable.labelled("excludedConnectorNames", _names.getExcluded()));
```

Comment:
See above, use "includedConnector" only (drop "Names").

Reply:
And no, there is no need for `Dumpable.labelled()`.
Use `DumpableCollection` instead.

Reply:
Nah - DumpableCollection introduces an extra level of nesting when it is not needed.

Reply:
For example the code:
```java
        System.err.println(DumpableCollection.from("label", "item").dump());
```
results in the needlessly complex:
```
label size=1
+> item
```

while
```java
        System.err.println(Dumpable.labelled("label", "item").dump());
```
results in the simpler
```
label:item
```

In this case the simpler form gives:
```
InetAccessHandler@36d585c{STARTED} - STARTED
+= AbstractHandler@671a5887{STARTED} - STARTED
+> included:org.eclipse.jetty.util.InetAddressSet@0(size=0)
+> excluded:org.eclipse.jetty.util.InetAddressSet@0(size=0)
+> includedConnectorNames:java.util.HashSet@0(size=0)
+> excludedConnectorNames:java.util.HashSet@0(size=0)
```
instead of the confusing:
```
InetAccessHandler@36d585c{STARTED} - STARTED
+= AbstractHandler@671a5887{STARTED} - STARTED
+> included size=1
|  +> org.eclipse.jetty.util.InetAddressSet@0(size=0)
+> excluded size=1
|  +> org.eclipse.jetty.util.InetAddressSet@0(size=0)
+> includedConnectorNames size=1
|  +> java.util.HashSet@0(size=0)
+> excludedConnectorNames size=1
   +> java.util.HashSet@0(size=0)
```
which as size 1 for empty sets!

There is room for both methods

Reply:
OK cancel that... in this case they are collections so I should not use the `from` method. Creating new `DumpableCollection`s results is the much better:
```
InetAccessHandler@36d585c{STARTED} - STARTED
+= AbstractHandler@671a5887{STARTED} - STARTED
+> included size=1
|  +> 127.0.0.1
+> excluded size=0
+> includedConnectorNames size=0
+> excludedConnectorNames size=1
   +> http
```

I think the labelled is useful elsewhere... when collections are not involved.